src/vm/ceeload.cpp

```diff
@@ -3422,8 +3422,8 @@ BOOL Module::IsInSameVersionBubble(Module *target)
 {
     CONTRACT(BOOL)
```

Comment:
It is best to prefer `STANDARD_VM_CONTRACT` where possible. It means "regular code" with no special requirements. It saves you from all the contract troubles.

Reply:
Ok, thanks, let me change to that.